packages/paste-website/src/pages/components/tabs/index.mdx

```diff
@@ -85,20 +86,32 @@ export const getStaticProps = async () => {
 
 ### About Tabs
 
-The Tabs component allows a user to flip through multiple views within the same page.
+Tabs are labeled controls that allow users to switch between multiple views within a page. Tabs do not take the user to a new URL. To take users to a new URL, use [In Page Navigation](/components/in-page-navigation).
+
+Panels within a set of Tabs should present different views of similar, parallel information. For example, [these Twilio docs](https://www.twilio.com/docs/usage/api) contain a block of code on the right side of the page with Tabs, allowing users to view the code in different languages. [Reddit also uses tabs](https://www.reddit.com/t/technology/), switching the user’s view between **Posts**, **Communities**, and **Related Topics**. The Tabs in both of these examples control views; they don’t direct to a new page. Conversely, [the Github website](https://github.com/twilio-labs/paste) demonstrates how some interactions may seem like Tabs, but are actually in-page navigation. Clicking on one of the sections at the top, like **Code** or **Pull Requests**, changes the URL.
+
+Choose Tabs when users don’t need to see content from multiple views simultaneously. When thinking through what type of content to put in your Tabs, remember that the content should be parallel across the board. What lives in one Tab should be a different view of what lives in another Tab.
```

Comment:
suggestion: having added my suggestion below, I now realise that names are hard and might confuse people. Would is be worth adding some kind of glossary of terms up here:

- *Tabset:* a composition of a Tablist and Tabpanels
- *Tablist:* a collection of Tabs
- *Tab:* clickable element that causes a selection within the Tabset
- *Tabpanel:* contains the content associated with the selected Tab